susemanager-utils/susemanager-sls/modules/roster/uyuni.py

```diff
@@ -89,7 +89,9 @@ def __init__(self, db_config, uyuni_roster_config):
                 )
             )
 
-        log.trace("db_connect string: %s", self.db_connect_str)
+        log.trace("db_connect dbname: %s", db_config['db'])
+        log.trace("db_connect   user: %s", db_config['user'])
+        log.trace("db_connect   host: %s", db_config['host'])
```

Comment:
Just one nitpick. Let's keep "more modern" double quote style:
```suggestion
        log.trace("db_connect dbname: %s", db_config["db"])
        log.trace("db_connect   user: %s", db_config["user"])
        log.trace("db_connect   host: %s", db_config["host"])
```
Other than that looks good to me :+1: